### ISSUE TITLE
Support zone.js 0.10.0

### ISSUE THREAD

**Issue by fr0** [angular/angular#31896]
# 🚀 feature request

### Relevant Package

 This feature request is for @angular/core

```
warning " > @angular/core@8.1.3" has incorrect peer dependency "zone.js@~0.9.1".
```

0.10.0 contains multiple [bug fixes](https://github.com/angular/angular/blob/master/packages/zone.js/CHANGELOG.md) that will benefit many Angular users.

**Comment by JiaLiPassion** [comment]
We will have a new release, after that I will update the dependency, thanks.

**Comment by vladimiry** [comment]
@fr0, 0.10.0 seems not to be quite ready for production use, at least due to the https://github.com/angular /angular/pull/31866

**Comment by rafa-as** [comment]
Some news here?

**Comment by JiaLiPassion** [comment]
The new release will be ready very soon

**Comment by rafa-as** [comment]
Thanks @JiaLiPassion !

**Comment by UnleashMayhem** [comment]
Any news when angular will be updated to support this?

**Comment by JiaLiPassion** [comment]
The PR have been merged, please wait for a little more time.

**Comment by peterbsmith2** [comment]
@JiaLiPassion is it safe to use `zone.js@0.10.x` now in Angular 8?

**Comment by JiaLiPassion** [comment]
@peterbsmith2 , sure, it is safe to use `zone.js@0.10.2` now.

**Comment by acohenOT** [comment]
The warning still present as of Angular 8.2.5.

`npm WARN @angular/core@8.2.5 requires a peer of zone.js@~0.9.1 but none is installed. You must install peer dependencies yourself.`

**Comment by AlejandroFlorin** [comment]
Still getting the warning in 8.2.6

**Comment by JiaLiPassion** [comment]
I will check it, thanks!

**Comment by jpike88** [comment]
@JiaLiPassion this warning has been going on for months, and there is no clear reason as to why it's being delayed. Can you please shed some light on this?

**Comment by JiaLiPassion** [comment]
@jpike88,  the zone.js 0.10.x has been merged to `angular-cli`, so if you use `angular/cli@next` (currently is `9.0.0-next.8`), the zone.js will be `0.10.2`, and the warning will be gone.

**Comment by fr0** [comment]
Is there not going to be support for zone.js 0.10.0 with Angular 8.X?

**Comment by jpike88** [comment]
would be nice... I've been using 0.10.2 for ages and things seem to work perfectly for me

**Comment by jpike88** [comment]
Still getting this warning. Can someone give an update

**Comment by sharvesh** [comment]
upgraded to angular 8 and installed primeng, 
got  "primeng@9.0.0 requires a peer of zone.js@^0.10.2" .. fixed that ended up getting  "@angular/core@8.2.14 requires a peer of zone.js@~0.9.1"

**Comment by jelbourn** [comment]
This was resolved with Angular v9
